### DISCUSSION TITLE
posts/migrating-blog

### DISCUSSION THREAD

**Discussion by giscus** [TechWatching/techwatching.dev#21] [Announcements]
# posts/migrating-blog

https://www.techwatching.dev/posts/migrating-blog

**Comment by loic-sharma** [comment]
This is really cool! Thanks for sharing :)

**Reply by jbpaux** [reply]
Super interresting, I'll copy it 😁

**Comment by ankushjain358** [comment]
Super cool stuff man...I will also move my blog from Umbraco to Statiq. Excellent guide.

**Comment by thbst16** [comment]
Alexandre, this blog post (and having the source on GitHub) was the most helpful guide in my journey to port my blog to Statiq. I added a [couple of specific configuration options](https://beckshome.com/2022/09/beckshome-on-statiq) that I needed to make above and beyond what I found widely documented. Thanks for taking the time to write this up.

**Reply by TechWatching** [reply]
You're welcome! Thanks for the comment and the mention in your article. Good idea to have written down and explained your specific configuration options.
